main.rb

```diff
@@ -51,6 +58,23 @@ class Saying
   { saying: Saying.all.sample }.to_json
 end
 
+get '/slackit/?' do
+  saying = Saying.where(:crap_count.lt => 3).sample
+  if saying.inc(:crap_count, 1)
+    @attachments = [
+      {
+        author_name: saying.who,
+        color: 'good',
+        title: 'saywut!',
+        text: saying.wut
+      }
+    ]
+    Slack::Post.post_with_attachments saying.who,
+    @attachments, '#lemon-party-room'
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.